packages/shared/dao/sagas.ts

```diff
@@ -167,7 +167,8 @@ function* selectRealm() {
 
   if (!realm) debugger
 
-  return realm
+  // return realm
+  return { hostname: 'https://peer-testing.decentraland.org', protocol: 'v2', serverName: 'testing' }
```

Comment:
`?realm=v2~peer-testing.decentraland.org` in the URL does the trick

Reply:
I know, but for some reason, it was still changing it back